src/officehours_ui/urls.py

```diff
@@ -16,4 +17,6 @@
     path('add_queue/', SpaView.as_view(), name='add_queue'),
     path('auth/<backend_name>/', AuthPromptView.as_view(), name='auth_prompt'),
     path('callback/<backend_name>/', auth_callback_view, name='auth_callback'),
+    path("robots.txt", SpaView.as_view(template_name="robots.txt", content_type="text/plain")),
```

Comment:
`robots.txt` could also probably also be defined in views.py without a separate file. If you didn't want to do that I'd _probably_ use the TemplateView as SpaView is just a special view to redirect to the SPA front page. 

```
def robots_txt(request):
    content = "User-agent: *\nDisallow: /"
    return HttpResponse(content, content_type="text/plain")
```

And then referenced directly.
`    path('robots.txt', views.robots_txt, name='robots_txt'),`

Reply:
That kind of view would work, or I think I'd just serve a static file using `RedirectView` like `favicon.ico`. It seems like since `robots.txt` implies it's a file, it's a bit more straightforward to provide one.

Reply:
Yeah, that seems like a good idea too. Looks to work.